modules/src/ics03_connection/msgs.rs

```diff
@@ -662,8 +744,8 @@ mod tests {
                 want_pass: false,
             },
         ]
-        .into_iter()
-        .collect();
+            .into_iter()
```

Comment:
Why the indentation here?

Reply:
I think it's fixed now but no idea. Just ran `cargo fmt`, didn't notice it.